yarn-project/cli/src/cmds/l1/index.ts

```diff
@@ -21,20 +21,22 @@ export function injectCommands(program: Command, log: LogFn, debugLogger: DebugL
       'Url of the ethereum host. Chain identifiers localhost and testnet can be used',
       ETHEREUM_HOST,
     )
-    .requiredOption('-pk, --private-key <string>', 'The private key to use for deployment', PRIVATE_KEY)
+    .option('-pk, --private-key <string>', 'The private key to use for deployment', PRIVATE_KEY)
```

Comment:
This can be optional if a mnemonic is provided (which is always true because there's a default).